src/electron/views/device-connect-view/components/android-setup/prompt-choose-device-step.tsx

```diff
@@ -61,22 +94,27 @@ export const PromptChooseDeviceStep = NamedFC<CommonAndroidSetupStepProps>(
                             ) : null;
                         }}
                         onRenderItemColumn={item => {
-                            return <p>{item.value}</p>;
+                            return (
+                                <DeviceDescription
+                                    className={styles.row}
+                                    {...item.metadata}
+                                ></DeviceDescription>
+                            );
                         }}
                     />
                 </>
             ),
             leftFooterButtonProps: {
                 text: 'Close',
-                onClick: _ => props.deps.closeApp(),
+                onClick: _ => this.props.deps.closeApp(),
```

Comment:
Just a general question: why did we have to specify `this.props` instead of `props` like previously?

Reply:
I did this because the component changed from a functional component to a class (to support state) - you can see some docs [here](https://reactjs.org/docs/state-and-lifecycle.html#converting-a-function-to-a-class).